test/represent-test.coffee

```diff
@@ -0,0 +1,159 @@
+                              "typeSpecification": {
+                                "name": "object",
+                                "nestedTypes": []
+                              },
+                              "attributes": []
+                            }
+                          },
+                          "sections": [
+                            {
+                              "class": "memberType",
+                              "content": [
+                                {
+                                  "content": {
+                                    "name": {
+                                      "literal": "id"
+                                    },
+                                    "description": "",
+                                    "valueDefinition": {
+                                      "values": [
+                                        {
+                                          "literal": "pavan",
+                                          "variable": false
+                                        }
+                                      ],
+                                      "typeDefinition": {
+                                        "typeSpecification": {
+                                          "name": null,
+                                          "nestedTypes": []
+                                        },
+                                        "attributes": []
+                                      }
+                                    },
+                                    "sections": []
+                                  },
+                                  "class": "property"
+                                }
+                              ]
+                            }
+                          ]
+                        },
+                        "class": "value"
+                      },
+                      {
+                        "content": {
+                          "description": "",
+                          "valueDefinition": {
+                            "values": [
+                              {
+                                "literal": "pkumar",
+                                "variable": false
+                              }
+                            ],
+                            "typeDefinition": {
+                              "typeSpecification": {
+                                "name": null,
+                                "nestedTypes": []
+                              },
+                              "attributes": []
+                            }
+                          },
+                          "sections": []
+                        },
+                        "class": "value"
+                      }
+                    ]
+                  }
+                ]
+              },
+              "class": "property"
+            }
+          ]
+        }
+      ]
+    }
```

Comment:
Changing the script from protagonist-experimental to drafter is failing a few tests. I didn't want to deal with it for now.